SWLOR.Game.Server/Service/SkillService/SkillType.cs

```diff
@@ -315,6 +321,8 @@ public class SkillAttribute : Attribute
         public bool IsShownInCraftMenu { get; set; }
         public CharacterType CharacterTypeRestriction { get; set; }
 
+        public CombatPointCategory XPType { get; set; } 
```

Comment:
With the change to include the suffix of "Type" on CombatPointCategoryType let's go ahead and rename XPType to CombatPointCategory or something similar to make it a little clearer. XPType could be confusing - one might think it refers to combat XP, RP XP, etc.